<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <widget class="QStackedWidget" name="s_principal">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="login">
       <layout class="QGridLayout" name="gridLayout">
        <item row="3" column="0">
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="1">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="3" column="1">
         <widget class="QGroupBox" name="g_login">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="title">
           <string>Login</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item>
            <widget class="QLineEdit" name="loginInput">
             <property name="placeholderText">
              <string>Usuário</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="passwordInput">
             <property name="echoMode">
              <enum>QLineEdit::Password</enum>
             </property>
             <property name="placeholderText">
              <string>Senha</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="loginButton">
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Entrar</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="3" column="2">
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="4" column="1">
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="0" column="2">
         <widget class="QLabel" name="status">
          <property name="font">
           <font>
            <pointsize>12</pointsize>
           </font>
          </property>
          <property name="text">
           <string>TextLabel</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="aluno">
       <layout class="QGridLayout" name="gridLayout_2">
        <item row="0" column="0">
         <widget class="QTabWidget" name="t_aluno">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="a_perfil">
           <attribute name="title">
            <string>Perfil</string>
           </attribute>
           <widget class="QPushButton" name="logout_aluno">
            <property name="geometry">
             <rect>
              <x>150</x>
              <y>200</y>
              <width>80</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Sair</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="a_prova">
           <attribute name="title">
            <string>Fazer Prova</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="professor">
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTabWidget" name="t_professor">
          <property name="tabShape">
           <enum>QTabWidget::Rounded</enum>
          </property>
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="t_perfil">
           <attribute name="title">
            <string>Perfil</string>
           </attribute>
           <widget class="QLabel" name="p_foto">
            <property name="geometry">
             <rect>
              <x>30</x>
              <y>30</y>
              <width>128</width>
              <height>128</height>
             </rect>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel{
border-image: url(:/images/foto_login.png);
}</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
           <widget class="QLabel" name="p_nome">
            <property name="geometry">
             <rect>
              <x>10</x>
              <y>160</y>
              <width>171</width>
              <height>31</height>
             </rect>
            </property>
            <property name="text">
             <string>TextLabel</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
            <property name="textInteractionFlags">
             <set>Qt::NoTextInteraction</set>
            </property>
           </widget>
           <widget class="QPushButton" name="logout_professor">
            <property name="geometry">
             <rect>
              <x>60</x>
              <y>200</y>
              <width>64</width>
              <height>21</height>
             </rect>
            </property>
            <property name="text">
             <string>Sair</string>
            </property>
           </widget>
          </widget>
          <widget class="QWidget" name="t_editarprova">
           <attribute name="title">
            <string>Criar Prova</string>
           </attribute>
           <layout class="QGridLayout" name="gridLayout_4">
            <item row="2" column="2">
             <widget class="QGroupBox" name="g_enunciado">
              <property name="title">
               <string>Enunciado</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <item>
                <widget class="QTextEdit" name="t_enunciado">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="4" column="2">
             <widget class="QGroupBox" name="g_resposta">
              <property name="title">
               <string>Resposta</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QTextEdit" name="t_resposta">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                 <property name="html">
                  <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="0" rowspan="5">
             <widget class="QGroupBox" name="g_questoes">
              <property name="title">
               <string>Questões</string>
              </property>
              <layout class="QGridLayout" name="gridLayout_3">
               <item row="0" column="1" colspan="2">
                <widget class="QGroupBox" name="g_questoes_in">
                 <property name="title">
                  <string/>
                 </property>
                 <layout class="QVBoxLayout" name="verticalLayout_2">
                  <item>
                   <widget class="QListWidget" name="lista_questoes"/>
                  </item>
                  <item>
                   <widget class="QPushButton" name="remover_questao">
                    <property name="text">
                     <string>Remover Questão</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="adicionar_questao">
                    <property name="text">
                     <string>Nova Questão</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item row="1" column="1" colspan="2">
                <widget class="QPushButton" name="busca_bd">
                 <property name="acceptDrops">
                  <bool>false</bool>
                 </property>
                 <property name="layoutDirection">
                  <enum>Qt::LeftToRight</enum>
                 </property>
                 <property name="text">
                  <string>Buscar questões no banco de dados</string>
                 </property>
                 <property name="checkable">
                  <bool>false</bool>
                 </property>
                 <property name="flat">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item row="2" column="1" colspan="2">
                <widget class="QPushButton" name="gerar_prova">
                 <property name="text">
                  <string>Gerar Prova</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="1" column="2">
             <widget class="QGroupBox" name="g_titulo">
              <property name="title">
               <string>Título</string>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <item>
                <widget class="QTextEdit" name="t_titulo">
                 <property name="readOnly">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item row="0" column="2">
             <widget class="QComboBox" name="c_tipo">
              <item>
               <property name="text">
                <string>Multipla Escolha</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Programação</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>Discursiva</string>
               </property>
              </item>
             </widget>
            </item>
           </layout>
           <zorder>g_enunciado</zorder>
           <zorder>g_resposta</zorder>
           <zorder>g_questoes</zorder>
           <zorder>c_tipo</zorder>
           <zorder>g_titulo</zorder>
          </widget>
          <widget class="QWidget" name="t_aplicarprova">
           <attribute name="title">
            <string>Aplicar Prova</string>
           </attribute>
          </widget>
          <widget class="QWidget" name="t_criarquestao">
           <attribute name="title">
            <string>Criar Questão</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>loginInput</tabstop>
  <tabstop>passwordInput</tabstop>
  <tabstop>loginButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
